js/coinsbit.js

```diff
@@ -0,0 +1,799 @@
+            'countries': [ 'EE' ], // Estonia
+            'rateLimit': 1000, // No defaults known
+            'version': 'v1',
+            'certified': false,
+            'has': {
+                'addMargin': undefined,
+                'cancelAllOrders': undefined,
+                'createOrder': true,
+                'cancelOrder': undefined,
+                'createMarketOrder': undefined,
+                'fetchBalance': true,
+                'fetchCanceledOrders': undefined,
+                'fetchClosedOrders': undefined,
+                // 'fetchCurrencies' doesn't exist, and will be 'emulated'
+                'fetchDepositAddress': undefined,
+                'fetchDeposits': undefined,
+                'fetchFundingRateHistory': undefined,
+                'fetchMarkets': true,
+                // 'fetchMarketsByType': <-- i think fetchMarketsByType makes things complex, and might be totally removed from ccxt. There should be one 'fetchMarkets' imho. you can see another comment related to this matter under 'handleMarketTypeAndParams' function down below.
+                'fetchMyTrades': undefined,
+                'fetchOHLCV': true,
+                'fetchOpenOrders': undefined,
+                'fetchOrder': undefined,
+                'fetchOrderBook': true,
+                'fetchOrderTrades': undefined,
+                'fetchPosition': undefined,
+                'fetchPositions': undefined,
+                'fetchStatus': false,
+                'fetchTicker': true,
+                'fetchTickers': true,
+                'fetchTime': false,
+                'fetchTrades': true,
+                'fetchWithdrawals': undefined,
+                'reduceMargin': undefined,
+                'setLeverage': undefined,
+                'withdraw': undefined,
+            },
+            'timeframes': {
+                '1m': 60, // works for max 12 hours diapason (between from-to)
+                '30m': 1800, // for max 24 hours diapason
+                '1h': 3600, // for max 7 days diapason
+                '1d': 86400, // for any diapason
+            },
+            'urls': {
+                'logo': '  <<< TODO >>>   https://upload.wikimedia.org/wikipedia/commons/8/8c/Coinsbit.png  ',
+                'api': {
+                    'public': 'https://coinsbit.io',
+                    'private': 'https://coinsbit.io',
+                },
+                'www': 'https://coinsbit.io/',
+                'doc': [
+                    'https://www.notion.so/coinsbitwsapi/API-COINSBIT-WS-API-COINSBIT-cf1044cff30646d49a0bab0e28f27a87',
+                    'https://github.com/Coinsbit-connect/api/wiki/API',
+                ],
+                'fees': [
+                    'https://coinsbit.io/fee-schedule',
+                ],
+                'referral': '  <<< TODO >>>   ',
+            },
+            'api': {
+                'public': {
+                    'get': {
+                        'markets': 1,
```

Comment:
it's uneccesary to put `markets': '1'` since 1 is already the default cost for the endpoints

Reply:
I would keep it explicit in all exchanges from now on.